recipe/meta.yaml

```diff
@@ -45,8 +44,12 @@ about:
   home: https://pycodestyle.readthedocs.io/
   license: MIT
   license_file: LICENSE
+  license _url: https://github.com/PyCQA/pycodestyle/blob/main/LICENSE
```

Comment:
There is an extra space in `license _url:`: However, we should remove `license_url` and `doc_source_url` if `license_file` and `doc_url` are present per the updated recipe guidelines.

Reply:
Got it! Made the necessary change.